x-pack/plugins/cases/server/client/cases/bulk_update.test.ts

```diff
@@ -1726,38 +1783,6 @@ describe('update', () => {
           `"Failed to update case, ids: [{\\"id\\":\\"mock-id-1\\",\\"version\\":\\"WzAsMV0=\\"}]: Error: Unauthorized"`
         );
       });
-
-      it('throws when user is not authorized to reopen case', async () => {
```

Comment:
why was this completely removed?

Reply:
Because it did not add value, this test and the one above do the same thing.

Reply:
Ah I see they both do `clientArgs.authorization.ensureAuthorized.mockRejectedValue(error);` ok, makes sense.